app/services/DeveloperService.scala

```diff
@@ -195,5 +195,5 @@ class DeveloperService @Inject()(appConfig: AppConfig,
     }
   }
 
-  private def combine[T](futures: Seq[Future[Seq[T]]]): Future[Seq[T]] = Future.reduce(futures)(_ ++ _)
+  private def combine[T](futures: List[Future[List[T]]]): Future[List[T]] = Future.reduceLeft(futures)(_ ++ _)
```

Comment:
Can this be tidied/in-lined? Passing around List-of-Future-of-List-of-Something feels a funky.

(Although I get that the change was only around using Lists instead of Seqs)

Reply:
Yeah it's a bit nasty but don't want to change how the code works too much right now.